assets/collections/comecando_com_git.json

```diff
@@ -4,6 +4,44 @@
   "description": "\"Git é um sistema de controle de versões distribuído, usado principalmente no desenvolvimento de software, mas pode ser usado para registrar o histórico de edições de qualquer tipo de arquivo. O Git foi inicialmente projetado e desenvolvido por Linus Torvalds para o desenvolvimento do kernel Linux, mas foi adotado por muitos outros projetos.\" - Wikipedia, 2021.",
   "category": "Versionamento",
   "tags": ["git", "versionamento"],
+  "contributors": [
```

Comment:
Please replace the collections mocked data with matuella metadata:

```json
{
  "name": "matuella",
  "url": "https://github.com/matuella",
  "imageUrl": "https://avatars.githubusercontent.com/u/13678134?v=4"
}
```

Reply:
👍  ok. But won't these collections be created again with the correct contributors?

Reply:
@ggirotto I replaced all them with (and only) matuella's metadata...